--- Makefile.orig	2015-03-21 11:43:21.000000000 +0000
+++ Makefile	2015-03-21 11:46:05.000000000 +0000
@@ -12,16 +12,14 @@
 MAINTAINER=	ports@FreeBSD.org
 COMMENT=	SixXS IPv6 TIC+ tunnel broker heartbeat client
 
-LIB_DEPENDS=	libgnutls-openssl.so:${PORTSDIR}/security/gnutls
-
 LICENSE=	BSD3CLAUSE
 LICENSE_FILE=	${WRKSRC}/doc/LICENSE
 
 USE_RC_SUBR=	sixxs-aiccu
 USES=		gmake
 
-CFLAGS+=	${PTHREAD_CFLAGS} -D_REENTRANT -DAICCU_GNUTLS -I${LOCALBASE}/include
-LDFLAGS+=	${PTHREAD_LIBS} -L${LOCALBASE}/lib -lgnutls
+CFLAGS+=	${PTHREAD_CFLAGS} -D_REENTRANT -I${LOCALBASE}/include
+LDFLAGS+=	${PTHREAD_LIBS} -L${LOCALBASE}/lib 
 MAKE_ARGS+=	CC="${CC}" LDFLAGS="${LDFLAGS}"
 WRKSRC=		${WRKDIR}/aiccu
 BUILD_WRKSRC=	${WRKDIR}/aiccu/unix-console
